README.md

```diff
@@ -6,5 +6,27 @@ Thomas Bot
 Thomas Bot is the friendly Discord bot! It hangs around in the official IT Factory Discord server.
 It helps teachers doing their job and students also... sometimes... 
 
+## Running locally
+
+### Build manually
+0. Make sure the [Go toolchain](https://golang.org/doc/install) is installed and working.
+1. Make sure the required dependencies are installed:
+    - On Ubuntu and other Debian-based distros, install the following packages: `libsox-dev libsdl2-dev portaudio19-dev libopusfile-dev libopus-dev curl`.
+    - On Arch-based distros, install the following packages: `libsoxr sdl portaudio opusfile`.
+    - On Windows, you're on your own for now :)
+2. Build the project using `go build`.
```

Comment:
2 and 3 for development purposed can be consolidated into `THOMASBOT_TOKEN={TOKEN} go run ./`

Reply:
Done